lib/proxy.js

```diff
@@ -116,16 +116,16 @@ function makeSimpleProxy(host, port, options, pluginID, serviceName) {
       res2.pipe(res1);
     });
     req2.on('error', (e) => {
-      proxyLog.warn('Callservice: Service call failed.');
-      console.warn(e);
-      res1.status(500).send(`Unable to complete network request to ${host}:${port}: `
+      proxyLog.warn('ZWED0040W'); //proxyLog.warn('Callservice: Service call failed.');
+      console.warn("ZWED0040W - ", e);
```

Comment:
shouldn't this just be 1 line:
roxyLog.warn('ZWED0040W', e);

Reply:
Addressed.